tests/ipfwd/test_nhop_count.py

```diff
@@ -225,12 +225,39 @@ def combinations(iterable, r):
 
 def loganalyzer_ignore_regex_list():
     ignore = [
-        ".*Unaccounted_ROUTE_ENTRY_TABLE_entries.*"
+        ".*Unaccounted_ROUTE_ENTRY_TABLE_entries.*",
+        ".*ERR swss#orchagent: :- addAclTable: Failed to.*",
+        ".*ERR swss#orchagent: :- create: create status:.*",
+        ".*ERR syncd#syncd: [none] SAI_API_ACL:brcm_sai_dnx_create_acl_table:338 create table.*",
+        ".*ERR syncd#syncd: [none] SAI_API_ACL:_brcm_sai_dnx_create_acl_table:7807 field group.*",
+        ".*ERR syncd#syncd: :- processQuadEvent: attr: SAI_ACL_TABLE_ATTR_ACL_BIND_POINT_TYPE_LIST:.*",
+        ".*ERR syncd#syncd: :- processQuadEvent: attr: SAI_ACL_TABLE_ATTR_ACL_STAGE:.*",
+        ".*ERR syncd#syncd: :- processQuadEvent: attr: SAI_ACL_TABLE_ATTR_FIELD_ACL_IP_TYPE:.*",
+        ".*ERR syncd#syncd: :- processQuadEvent: attr: SAI_ACL_TABLE_ATTR_FIELD_ACL_RANGE_TYPE:.*",
+        ".*ERR syncd#syncd: :- processQuadEvent: attr: SAI_ACL_TABLE_ATTR_FIELD_DSCP:.*",
+        ".*ERR syncd#syncd: :- processQuadEvent: attr: SAI_ACL_TABLE_ATTR_FIELD_DST_IP:.*",
+        ".*ERR syncd#syncd: :- processQuadEvent: attr: SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6:.*",
```

Comment:
This might be temporary error messages do we need to ignore list ?

Reply:
The test gets reported as failure. Messages seem specific and not related to nhop resource so it shouldn't affect other SKUs. I'll update when it's been taken care of.